packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -1199,13 +1226,14 @@ describeIf(!process.env.TEST_SKIP_MSSQL)('SQL Server', () => {
   })
 })
 
-// TODO: Windows: tests fail on Windows, introspected schema differs from snapshots.
-// TODO: macOS: disabled on CI because it fails with timeout. Somehow jest.setTimeout
-// doesn't seem to work in this test case particularly.
-describeIf(process.platform !== 'win32' && !isMacOrWindowsCI)('MongoDB', () => {
+describeIf(!process.env.TEST_SKIP_MONGODB)('MongoDB', () => {
```

Comment:
```suggestion
// TODO: Windows: tests fail on Windows, introspected schema differs from snapshots.
// errors with https://github.com/prisma/prisma/actions/runs/3873515447/jobs/6603629539#step:13:833
// Received promise rejected instead of resolved
// Rejected to value: [Error: P1012 Introspection failed as your current Prisma schema file is invalid·
// Please fix your current schema manually (using either prisma validate or the Prisma VS Code extension to 
// understand what's broken and confirm you fixed it), and then run this command again.
// Or run this command with the --force flag to ignore your current schema and overwrite it. 
// All local modifications will be lost. ]
describeIf(process.platform !== 'win32' && !process.env.TEST_SKIP_MONGODB)('MongoDB', () => {
```